src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/translator_fallback.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:framework="http://symfony.com/schema/dic/symfony"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/symfony http://symfony.com/schema/dic/symfony/symfony-1.0.xsd">
+
+    <framework:config secret="s3cr3t">
+        <framework:translator enabled="true">
+            <framework:fallback>en</framework:fallback>
+            <framework:fallback>fr</framework:fallback>
+        </framework:translator>
+    </framework:config>
+</container>
```

Comment:
This file does not seem to be used anywhere in the tests. Have you forgot to commit them? Tests should also be done for other formats like YAML.